docs/eventing/debugging/example.yaml

```diff
@@ -99,9 +99,7 @@ spec:
     spec:
       containers:
         - name: user-container
-          # This corresponds to
-          # https://github.com/knative/eventing-sources/blob/v0.2.1/cmd/message_dumper/dumper.go.
-          image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/message_dumper@sha256:ab5391755f11a5821e7263686564b3c3cd5348522f5b31509963afb269ddcd63
+          image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display@sha256:ab5391755f11a5821e7263686564b3c3cd5348522f5b31509963afb269ddcd63
```

Comment:
I wouldn't change the debugging example. If you do, the MD also needs to change (not purely mechanical changes either).